wled00/FX_2Dfcn.cpp

```diff
@@ -276,8 +276,8 @@ void Segment::blur2D(uint8_t blur_x, uint8_t blur_y, bool smear) {
   if (!isActive()) return; // not active
   const unsigned cols = vWidth();
   const unsigned rows = vHeight();
-  uint32_t lastnew;
-  uint32_t last;
+  uint32_t lastnew = BLACK;
```

Comment:
These are actually not used when unintialised. `if (x>0)` takes care of that. Similar in FX_fcn.cpp